### DISCUSSION TITLE
Shall we list companies that supports WFH but don't base in China?

### DISCUSSION THREAD

**Discussion by Superskyyy** [LinuxSuRen/remote-jobs-in-china#21] [Ideas]
There are companies that support working remotely from China but do not currently base in China.

For example, [Tetrate](https://www.tetrate.io). I suppose there are more out there.

**Comment by LinuxSuRen** [comment]
Thanks for your suggestion. How about letting us make two lists here? One for those companies based in China, one for those have offices in China?

**Reply by Superskyyy** [reply]
> Thanks for your suggestion. How about letting us make two lists here? One for those companies based in China, one for those have offices in China?

Certainly! That would be best.
